packages/api-request-builder/src/default-params.js

```diff
@@ -68,6 +68,8 @@ export function getDefaultSearchParams(): ServiceBuilderDefaultParams {
  */
 export function setDefaultParams() {
   this.params.expand = getDefaultQueryParams().expand
+  
+  this.params.key = null
```

Comment:
Nice 👍 

Looking through this file, I'm also wondering if we are ever resetting all other params? For example, version can be set here https://github.com/commercetools/nodejs/blob/68c34e7a126d7a82cc867e99fab553ab77827ea7/packages/api-request-builder/src/default-params.js#L215

but it's not reset inside this function, `setDefaultParams()`